packages/bpk-component-link/src/BpkLink.js

```diff
@@ -60,15 +71,17 @@ BpkLink.propTypes = {
   className: PropTypes.string,
   onClick: PropTypes.func,
   blank: PropTypes.bool,
+  alternate: PropTypes.bool,
   white: PropTypes.bool,
 };
 
 BpkLink.defaultProps = {
   className: null,
   onClick: null,
   blank: false,
+  alternate: false,
```

Comment:
Could we add a `DEPRECATED:` comment here so that we know to remove `white` eventually

Reply:
done